### PR TITLE
[PIM-6887] Add a new page

### PR DESCRIPTION
**Pull request by pierallard**
Doc about "how to add a new page"

### CODE REVIEW

File: design_pim/guides/add_new_page.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+How to add a new page
+=====================
+
+In this document, we will see the basics on how to add a new page in Akeneo PIM. We will
+- create a new route to this page
+- fill this page with some information
+- create a link to this one
```
Comment (tamarasaurus) [inline design_pim/guides/add_new_page.rst]: to this page

File: design_pim/guides/add_new_page.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+        config:
+            pim/controller-registry:
+                controllers:
+                    acme_custom_index:
+                        module: acme/custom
+
+.. note::
+
+    The name of the controller route (`acme_custom_index`) your define on the controller registry has to be the same than the one declared in back-end routes.
```
Comment (tamarasaurus) [inline design_pim/guides/add_new_page.rst]: you defined in the controller registry has to be the same as the one

File: design_pim/guides/add_new_page.rst [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            module: pim/menu/tab
+            parent: pim-menu
+            position: 100
+            targetZone: mainMenu
+            config:
+                title: 'Custom'
+                iconModifier: iconCard
+                to: acme_custom_index
+
+To show your last changes, you have to dump new routes and recompute assets:
```
Comment (tamarasaurus) [inline design_pim/guides/add_new_page.rst]: To view your changes 

I think 'rebuild' makes more sense than 'recompute'

File: design_pim/guides/add_new_page.rst [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+.. code-block:: bash
+
+    $ bin/console pim:installer:dump-require-paths
+    $ bin/console pim:install:assets
+    $ npm run webpack-watch
```
Comment (tamarasaurus) [inline design_pim/guides/add_new_page.rst]: bin/console assets:install --symlink
yarn run webpack

File: design_pim/guides/add_new_page.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+You will have your new item on the main menu, and when you will on it, it will display an empty page. You can now add
```
Comment (tamarasaurus) [inline design_pim/guides/add_new_page.rst]: You will have your new item on the main menu, and when you click on it

File: design_pim/guides/add_new_page.rst [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    extensions:
+        acme-custom-index:
+          module: pim/common/simple-view
+          config:
+              template: pim/template/common/default-template
+
+Create the controller
+---------------------
+
+Then, create your controller and register it. This controller will simply render a complete form extension.
```
Comment (tamarasaurus) [inline design_pim/guides/add_new_page.rst]: what do you mean by 'complete' ?

File: design_pim/guides/add_new_page.rst [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+In this document, we will see the basics on how to add a new page in Akeneo PIM. We will
+- create a new route to this page
+- fill this page with some information
+- create a link to this page
+
+Create a new route
+------------------
+
+First, create a new back-end route to access to your new page
```
Comment (phaseinducer) [inline design_pim/guides/add_new_page.rst]: NO

Comment (phaseinducer) [inline design_pim/guides/add_new_page.rst]: Could be cool to explain why we need to do that in backend.

File: design_pim/guides/add_new_page.rst [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+        acme-custom-index:
+          module: pim/common/simple-view
+          config:
+              template: pim/template/common/default-template
+
+Create the controller
+---------------------
+
+Then, create your controller and register it. This controller will simply render a page form extension.
+This controller will simply load the extension declared above.
```
Comment (phaseinducer) [inline design_pim/guides/add_new_page.rst]: "This controller will simply" x2

File: design_pim/guides/add_new_page.rst [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+        config:
+            pim/controller-registry:
+                controllers:
+                    acme_custom_index:
+                        module: acme/custom
+
+.. note::
+
+    The name of the controller route (`acme_custom_index`) your defined in the controller registry has to be the same as the one you declared in back-end routes.
```
Comment (phaseinducer) [inline design_pim/guides/add_new_page.rst]: your -> you

Comment (phaseinducer) [reply design_pim/guides/add_new_page.rst]: `- routes`

File: design_pim/guides/add_new_page.rst [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+                title: 'Custom'
+                iconModifier: iconCard
+                to: acme_custom_index
+
+To view your changes, you have to dump new routes and rebuild assets:
+
+.. code-block:: bash
+
+    $ bin/console pim:installer:dump-require-paths
+    $ bin/console pim:install:assets
```
Comment (phaseinducer) [inline design_pim/guides/add_new_page.rst]: Can you try `bin/console pim:install:assets --symlink` and don't call `assets:install` ?

File: design_pim/guides/add_new_page.rst [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+    $ bin/console assets:install --symlink
+    $ yarn run webpack
+
+You will have your new item on the main menu, and when you click on it, it will display an empty page. You can now add
+some extensions under `acme-custom-index` to display your custom elements.
```
Comment (phaseinducer) [inline design_pim/guides/add_new_page.rst]: some extensions with `acme-custom-index` as parent ?

File: design_pim/guides/add_new_page.rst (hunk 10 above)
Comment (pierallard) [reply design_pim/guides/add_new_page.rst]: nope

**Pull request merged**